src/Components/MainMenu/withTitles.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../Styles/foundation/_variables.scss';
```

Comment:
because this is a new file and it's not imported in the mainmenu component this will be very difficult to consume. You'd have to import it beyond just the foundation scss. 

we should look at a different approach. we should wither just use this style all the time or do something where it is dynamically applied based on the depth of the items passed in.